lib/install/template.rb

```diff
@@ -11,10 +11,12 @@
 directory "#{__dir__}/config/loaders/core", "config/webpack/loaders"
 copy_file "#{__dir__}/config/.postcssrc.yml", ".postcssrc.yml"
 
-append_to_file ".gitignore", <<-EOS
+if File.exists?(".gitignore")
+  append_to_file ".gitignore", <<-EOS
```

Comment:
Yepp, I mean that we align that string block inline with `append_to_file` so, it's all in one line 👍  At the moment there is 2 whitespace for `append_to_file`, but none for string block